topicnet/demos/README.md

```diff
@@ -1 +1,27 @@
-### This folder contains Jupyter notebooks with working  examples of the code
\ No newline at end of file
+# Demo
+This section provides demonstrations of how to use this library in NLP tasks.
+
+1. [RTL-Wiki-Preprocessing](RTL-Wiki-Preprocessing.ipynb) --  notebook describing how to get a Wikipedia dataset and write data in VW format.
+
+2. [RTL-Wiki-Building-Topic-Mode](RTL-Wiki-Building-Topic-Model.ipynb) --  notebook with first steps to build topic model by consequently tuning its hyperparameters
+
+3. [Visualizing-Your-Model-Documents](Visualizing-Your-Model-Documents.ipynb) -- notebook providing a fresh outlook on unstructured document collection with the help of a topic model
+
+4. [20NG-Preprocessing](20NG-Preprocessing.ipynb) -- preparing data from a well-know 20 Newsgroups dataset
+
+5. [20NG-GenSim-vs-TopicNet](20NG-GenSim-vs-TopicNet.ipynb) -- a comparison between two topic models build by Gensim and TopicNet library. In the notebook we compare model topics by calculating their [UMass coherence measure](https://palmetto.demos.dice-research.org/) and using Jaccard measure to compare topic top-tokens diversity
```

Comment:
by calculating their UMass coherence measure with the help of [Palmetto](https://palmetto.demos.dice-research.org/)